lib_xua/src/core/mixer/mixer.xc

```diff
@@ -409,6 +409,9 @@ static inline void do_output_volume_control(int out_ch_index)
 #pragma unsafe arrays
 static void mixer1(chanend c_host, chanend c_mix_ctl, chanend ?c_mixer2, chanend c_audio)
 {
+    int cmd_pending_flag = 0;
+    int pending_cmd[3]; // command + a max of 2 extra values
```

Comment:
You could merge these two vars I guess, pending_cmd[0] ! = 0 for example

I think a struct would be better for this (and self documenting to an extent)

Reply:
what if 0 is a valid command? I think its safer having a separate pending flag?